cypress/plugins/index.js

```diff
@@ -212,6 +212,15 @@ module.exports = async (on, config) => {
     checkFileExists(filePath) {
       return fs.existsSync(filePath);
     },
+    loadJSFile(filePath) {
+      console.log('filepath 2652', filePath);
+      try {
+        const resolvedPath = path.resolve(__dirname, '..', filePath);
+        return require(resolvedPath);
+      } catch (error) {
+        return null;
```

Comment:
Curious about returning null instead of throwing an error? If null was a design decision could we at least print an error log?